packages/browser/src/plugins/in-app-plugin/__tests__/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import { Analytics } from '../../../core/analytics'
+import { Plugin } from '../../../core/plugin'
+import { pageEnrichment } from '../../page-enrichment'
+import { customerio, CustomerioSettings } from '../../customerio'
+import { InAppPlugin, InAppPluginSettings } from '../'
+import cookie from 'js-cookie'
+import Gist from 'customerio-gist-web'
+
+jest.mock('unfetch', () => {
+  return jest.fn()
+})
+
+describe('Customer.io In-App Plugin', () => {
+  let options: CustomerioSettings
+  let settings: InAppPluginSettings
+  let analytics: Analytics
+  let inAppPlugin: Plugin
+  let cio: Plugin
+
+  beforeEach(async () => {
+    jest.resetAllMocks()
+    jest.restoreAllMocks()
+
+    options = { apiKey: 'foo' }
```

Comment:
Some of these variables are only used here. Can you declare them as const inside the beforeEach?

My reasoning is that, at first glance, I expect a variable declared inside a describe to be used on a test somewhere.